logger.h

```diff
@@ -108,6 +109,14 @@ struct logentry_item_store {
     char key[];
 };
 
+struct logentry_deletion {
+    int nbytes;
```

Comment:
We might want to track the `cmd`, because deletes can be triggered from different commands.